static/js/tippyjs.js

```diff
@@ -366,4 +366,33 @@ export function initialize() {
         // Avoid inheriting `position: relative` CSS on the stream sorter widget.
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: "#toggle-private-messages-section-icon",
+        onShow(instance) {
+            const collapse_private_messages_tooptip = $t({
+                defaultMessage: "Collapse private messages",
+            });
+            const expand_private_messages_tooptip = $t({defaultMessage: "Expand private messages"});
```

Comment:
It seems cleaner to just inline these variables?

Reply:
Umm, I'm not sure if we 'll be able to do so, as we need to update the tippy dynamically.

Reply:
Please let me know if there's a way to do so.

Reply:
I don't understand -- you can call pass `$t(...)` to the Tippy functions inside the if/else blocks without defining a variable.

Reply:
Ohh, I think I misunderstood your comment; sorry for that, Updated the PR!